### ISSUE TITLE
It's not possible to not have any SLA on a ticket

### ISSUE THREAD

**Issue by ankush** [frappe/helpdesk#1598]
There are legitimate use cases where SLA shouldn't apply on a particular ticket. Right now if there's no SLA it fails with "internal server error"

Steps to reproduce:
- Configure SLAs such that a particular ticket might not have any applicable SLA. 
- Try to modify the ticket.

**Comment by ssiyad** [comment]
https://github.com/frappe/helpdesk/pull/1600

**Comment by ankush** [comment]
This still doesn't work. Just use it once.

**Comment by RitvikSardana** [comment]
The solution should be IMO for now is system should enforce one default SLA, what do you thing @ankush ?

**Comment by ankush** [comment]
> The solution should be IMO for now is system should enforce one default SLA,

Nope. That's the point of this issue, we don't want unnecessary SLA on certain tickets. Showing misleading SLA times to agent and customer is wrong.

**Comment by ankush** [comment]
The hack we are using internally is add 1 year as SLA time, that has outlived it's "temporary fix" status, it should be fixed in core.
